src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1444,6 +1504,38 @@ components:
       required:
         - status
 
+    ArchiveGoalRequest:
+      title: ArchiveGoalRequest
+      type: object
+      description: A request to archive an individual Goal.
+      properties:
+        goalReference:
```

Comment:
feels a bit weird to duplicate the goal reference between URL and body but prison number is only in the URL. again, did it this way to match existing pattern